CMakeLists.txt

```diff
@@ -854,24 +854,54 @@ set(wrapper_test_scripts
 	test-fake-ssh test-line-buffer test-svn-fe test-tool)
 
 
-foreach(script ${wrapper_scripts})
-	file(STRINGS ${CMAKE_SOURCE_DIR}/wrap-for-bin.sh content NEWLINE_CONSUME)
-	string(REPLACE "@@BUILD_DIR@@" "${CMAKE_BINARY_DIR}" content "${content}")
-	string(REPLACE "@@PROG@@" "${script}${EXE_EXTENSION}" content "${content}")
-	file(WRITE ${CMAKE_BINARY_DIR}/bin-wrappers/${script} ${content})
-endforeach()
+if(NOT DEFINED CI_BUILD)
 
-foreach(script ${wrapper_test_scripts})
+	foreach(script ${wrapper_scripts})
+		file(STRINGS ${CMAKE_SOURCE_DIR}/wrap-for-bin.sh content NEWLINE_CONSUME)
+		string(REPLACE "@@BUILD_DIR@@" "${CMAKE_BINARY_DIR}" content "${content}")
```

Comment:
Here, the only difference between `CI_BUILD` vs non-`CI_BUILD` is that we replace `@@BUILD_DIR@@` by a path relative to `GIT_EXEC_PATH` rather than a hard-coded, absolute path.

I could imagine that we will want this in general, not just in `CI_BUILD`.

The only thing that we need to be careful about is to use the `cygpath` tool only on Windows.